src/spid/spid.ts

```diff
@@ -44,6 +48,28 @@ import {
 } from "../utils/saml";
 import { getMetadataTamperer } from "../utils/saml";
 
+export const SpidUser = t.intersection([
+  t.interface({
+    // the following values may be set
+    // by the calling application:
+    // authnContextClassRef: SpidLevel,
+    // issuer: Issuer
+    getAssertionXml: t.Function
+  }),
+  t.partial({
+    email: EmailString,
+    familyName: t.string,
+    fiscalNumber: FiscalCode,
+    mobilePhone: NonEmptyString,
+    name: t.string,
+    nameID: t.string,
+    nameIDFormat: t.string,
+    sessionIndex: t.string
+  })
+]);
+
+export type SpidUser = t.TypeOf<typeof SpidUser>;
+
 // assertion consumer service express handler
 export type AssertionConsumerServiceT = (
```

Comment:
I have to apologize.. :-) this module is still dirty. A lot of theese definitions can now be imported from package `pagopa@io-spid-common` so it's pointless having them here. If you wish we may clean it as a part of another PR.

Reply:
Done :)